src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,63 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_attributes(session):
+    session.cable_length = nidcpower.CableLength.NI_STANDARD_TRIAXIAL_2M
+    session.lcr_short_custom_cable_compensation_enabled = False
+    assert session.cable_length == nidcpower.CableLength.NI_STANDARD_TRIAXIAL_2M
+    assert not session.lcr_short_custom_cable_compensation_enabled
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+@pytest.mark.parametrize(
+    "compensation_function",
+    [
+        nidcpower.Session.perform_lcr_open_custom_cable_compensation,
+        nidcpower.Session.perform_lcr_short_custom_cable_compensation,
+    ],
+)
+def test_perform_lcr_open_short_custom_cable_compensation(session, compensation_function):
+    compensation_function(session)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_custom_cable_compensation_data(session):
+    compensation_data = session.get_lcr_custom_cable_compensation_data()
+    session.configure_lcr_custom_cable_compensation(compensation_data)
+
+    session.configure_lcr_custom_cable_compensation(list(compensation_data))
+
+    session.configure_lcr_custom_cable_compensation(bytes(compensation_data))
+
+    with tempfile.NamedTemporaryFile(suffix='.bin', delete=False) as temp_file:
+        temp_file.write(compensation_data)
+    with open(temp_file.name, 'rb') as reopened_temp_file:
+        compensation_data_string = reopened_temp_file.read()
```

Comment:
Opening a file in `binary` mode and calling `read` will return `bytes` and not string. Did you mean to open in text mode?

Reply:
Nope, it was meant to be opened and read in binary mode as `tempfile.NamedTemporaryFile()` opens and writes the file in binary mode as well. I think `configure_lcr_custom_cable_compensation()` is not meant to work with `str` object directly as the same `str` may yield different `bytes` objects depending on the character encoding.

I have renamed the variable and the PR description to avoid confusion.